server/src/main/java/de/uftos/dto/ConstraintInstancesResponseDto.java

```diff
@@ -10,7 +14,45 @@
  * @param constraintInstances the instances of the constraints
  * @param displayNames        the display names of the arguments of the constraint instances
  */
-public record ConstraintInstancesResponseDto(@NotNull List<ConstraintInstance> constraintInstances,
+public record ConstraintInstancesResponseDto(@NotNull List<SlimInstance> constraintInstances,
                                              @NotNull
-                                             List<ConstraintArgumentDisplayName> displayNames) {
+                                             List<ConstraintArgumentDisplayName> displayNames,
+                                             @NotNull List<ConstraintParameter> parameters) {
+
+  /**
+   * Create a new ConstraintInstancesResponseDto instance with the standard entities
```

Comment:
```suggestion
   * Creates a new ConstraintInstancesResponseDto instance with the standard entities
```
I thought javaodc wasn't required for constructors?